release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -569,6 +565,10 @@ For more information on these new objects, see xref:../post_installation_configu
 
 For more information on converting existing ICSP YAML files to IDMS YAML files, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
 
+[id="ocp-4-13-toolbox-deprecation"]
+==== Toolbox is deprecated in {op-system}
+The toolbox library is deprecated and support will be removed from a future {product-title} release.
```

Comment:
Hum, it's not a library, it's a program (a script).

Reply:
fix is in #57902